### PR TITLE
Website: Table fix missing thead and move it before tbody

### PR DESCRIPTION
**Pull request by LeeWannacott**
#SemanticHTML. Please don't merge this yet. I need to fix this edge case (having multiple tbody that have no thead) in table-sort-js...

### CODE REVIEW

File: website/public/index.ejs.html [hunk 1]
```diff
@@ -683,8 +683,7 @@ <h2 id="linter-feature-comparison">Features</h2>
               <td class="good"><span aria-label="stable">✅</span></td>
             </tr>
           </tbody>
-
-          <tbody class="editors hide-in-mobile">
+          <thead class="hide-in-mobile">
```
Comment (strager) [inline website/public/index.ejs.html:686]: There can be only one thead per table [1], and the thead must not be after a tbody. There's a thead already on line 419, and there's a tbody immediately above this thead. So I don't think this HTML is legal.

[1] https://html.spec.whatwg.org/multipage/tables.html#the-thead-element

**Review by strager — CHANGES_REQUESTED** [review]

**Comment by strager** [comment]
What is the motivation for this change?

File: website/public/index.ejs.html (hunk 1 above)
Comment (LeeWannacott) [reply website/public/index.ejs.html:686]: legal in what sense? It works! you can have thead followed by tbody followed by thead followed by tbody. Maybe its the browser that makes it work even if its not in the spec?

Comment (LeeWannacott) [reply website/public/index.ejs.html:686]: > What is the motivation for this change?

I was trying to make the table on quick-lint sortable with firefox plugin. I know you don't care :crying_cat_face:

Comment (LeeWannacott) [reply website/public/index.ejs.html:686]: image
But you really don't!

Comment (LeeWannacott) [reply website/public/index.ejs.html:686]: Do I take the first row of each subsequent tbody and make it clickable then?

**Comment by LeeWannacott** [comment]
> What is the motivation for this change?
The last commit on quick-lint was a week ago.

**Comment by LeeWannacott** [comment]
I'll close. I guess I should support multiple tbodies differently even though it works : /

File: website/public/index.ejs.html (hunk 1 above)
Comment (strager) [reply website/public/index.ejs.html:686]: If a `tbody` has only one `tr`, and that `tr` only has `th`s, then I think you can treat that `tbody` as a `thead` for table-sort-js.

Comment (LeeWannacott) [reply website/public/index.ejs.html:686]: I think you mean if a `tbodys` first `tr` is only `th` `s` . Why would a tbody only have one tr big oof; this is complicated...

**Pull request closed without merging**